docs/DeveloperGuide.md

```diff
@@ -632,6 +631,22 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 * **Audit**: A record of all changes made in the system, including who made the changes and when.
 
 --------------------------------------------------------------------------------------------------------------------
+## **Appendix: Planned Enhancements**
+Team Size: 5
+1. **Update `add-patient` and `add-doctor` commands to check the date of birth and phone number input for validity**
+   1. Currently, both commands do not check whether the given data is a valid date of birth.
```

Comment:
I think valid DOB could be a bit confusing, maybe is "logical" is a better word here?